### PR TITLE
FED-2880 Fix Blink-based Microsoft Edge Browser Detection

### PR DESCRIPTION
**Pull request by aaronlademann-wf**
Currently, the modern Edge browser gets detected as `Chrome` - which is not ideal for logging purposes.

The browser - from a feature-set standpoint - literally IS `Chrome` - so it should remain being detected as such, but its name/version should reflect the application itself - not just the platform it is built atop of.

---

Also boy scouted some missing test coverage for the legacy (MSIE-based) Edge browser.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: test/browser_test.dart [hunk 1]
```diff
@@ -42,18 +44,42 @@ void main() {
...
+    test('Edge Chrome', () {
+      Browser.navigator = testChromeEdge();
+      browser = Browser.getCurrentBrowser();
+
+      expect(browser.name, 'Edge');
+      expect([browser.isChrome, browser.isEdgeChrome], everyElement(isTrue),
+          reason:
+              'Blink-based Edge browser should be detected as both Chrome and Edge '
+              'since feature support is based on Chrome, but should identify as Edge '
+              'for logging purposes');
```
Comment (aaronlademann-wf) [inline test/browser_test.dart:63]: This test ensures that any pre-existing feature-based business logic keying off of `isChrome` remains unchanged for Edge browsers.

**Review by robbecker-wf — DISMISSED** [review]

**Review by robbecker-wf — APPROVED** [review]
+10

**Comment by aaronlademann-wf** [comment]
@Workiva/release-management-pp

**Comment by robbecker-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM
